harvey/setrun.py

```diff
@@ -314,50 +312,116 @@ def setrun(claw_pkg='geoclaw'):
 
     # == setregions.data values ==
     regions = rundata.regiondata.regions
+
     # to specify regions of refinement append lines of the form
     #  [minlevel,maxlevel,t1,t2,x1,x2,y1,y2]
-    #Seadrift
-    rundata.gaugedata.gauges.append([1, -96.73, 28.38,
-                                     rundata.clawdata.t0,
-                                     rundata.clawdata.tfinal])
+    regions.append([1, 3, clawdata.t0, clawdata.tfinal, clawdata.lower[0], clawdata.upper[0], clawdata.lower[1], clawdata.upper[1]])
+    regions.append([1, 5, clawdata.t0, clawdata.tfinal, -99, -91, 26.5, 32]) # More specific domain f/ LaTex domain, minlevel = 1, maxlevel > 3, e.g. Can custommize based on time domain
 
-    #Port O'Connor
-    rundata.gaugedata.gauges.append([2, -96.38, 28.45,
-                                     rundata.clawdata.t0,
-                                     rundata.clawdata.tfinal])
 
-    #Aransas Wildlife Refuge
-    rundata.gaugedata.gauges.append([3, -96.75, 28.22,
-                                     rundata.clawdata.t0,
-                                     rundata.clawdata.tfinal])
 
-    #Port Aransas
-    rundata.gaugedata.gauges.append([4, -97.05, 27.84,
-                                     rundata.clawdata.t0,
-                                     rundata.clawdata.tfinal])
+	# Ruled rectangles -allows more custom definition of refinement regions
+    from clawpack.amrclaw.data import FlagRegion
+    flagregion = FlagRegion(num_dim=2)
+    flagregion.name = 'Region_LatexShelf'
+    flagregion.minlevel = 2
+    flagregion.maxlevel = 5
+    flagregion.t1 = 0.
+    flagregion.t2 = 1e9
+    flagregion.spatial_region_type = 2 # Ruled Rectangle
+    
 
-    #USS Lexington
-    rundata.gaugedata.gauges.append([5, -97.38, 27.80,
-				     rundata.clawdata.t0,
-                                     rundata.clawdata.tfinal])
+    from numpy import ma # masked arrays
+    from clawpack.visclaw import colormaps, plottools
+    from clawpack.geoclaw import topotools, marching_front
+    from clawpack.amrclaw import region_tools
+    import clawpack.geoclaw.topotools as topo
 
-    # Force the gauges to also record the wind and pressure fields
-    rundata.gaugedata.aux_out_fields = [4, 5, 6]
+    zmin = -60.
+    zmax = 40.
 
+    land_cmap = colormaps.make_colormap({ 0.0:[0.1,0.4,0.0],
```

Comment:
I do not see any plotting code below.  Is a bunch of this needed anymore?

Reply:
Lines 335 and 343-357 are not being used.  Also the imports 336-338 should be moved to the top of file.